sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRel.java

```diff
@@ -256,18 +283,24 @@ public void setup() {
     }
 
     @ProcessElement
-    public void processElement(@FieldAccess("row") Row row, OutputReceiver<Row> r) {
+    public void processElement(@FieldAccess("row") Row row, MultiOutputReceiver r) {
       assert se != null;
-      final Object[] v;
       try {
-        v = (Object[]) se.evaluate(new Object[] {row, CONTEXT_INSTANCE});
+        Object[] v = (Object[]) se.evaluate(new Object[] {row, CONTEXT_INSTANCE});
+        if (v != null) {
+          final Row output = toBeamRow(Arrays.asList(v), outputSchema, verifyRowValues);
+          r.get(rows).output(output);
```

Comment:
nit: This is a somewhat performance sensitive path, the extra `get` might slow things down. Is it possible to use `OutputReciever<Row>` for this case still?